src/main/java/org/reactome/addlinks/EnsemblBioMartUtil.java

```diff
@@ -66,7 +65,13 @@ private EnsemblBioMartUtil() {}
      * @return String, BioMart-formatted string.
      */
     public static String getBioMartSpeciesName(String speciesName) {
-        return speciesName.substring(0,1).toLowerCase() + speciesName.split(" ")[1];
+        String biomartSpeciesName = speciesName.substring(0,1).toLowerCase() + speciesName.split(" ")[1];
+        // May 2020 -- BioMart changed the Dog BioMart name from 'cfamiliaris' to 'clfamiliaris'.
+        // This can't be done automatically, so needs to be adjusted specifically.
+        if (speciesName.equals("Canis familiaris")) {
+            biomartSpeciesName = speciesName.substring(0,1).toLowerCase() + "l" + speciesName.split(" ")[1];
+        }
+        return biomartSpeciesName;
```

Comment:
Maybe for clarity in the speciesName processing, you could add these two temp variables to be used to form the `biomartSpeciesName` (renaming the parameter to something like `fullSpeciesName` or `genusAndSpeciesName`:

```
if (genusAndSpeciesName.equals("Canis familiaris")) {
   return "clfamiliaris"; // You could construct the name programatically, but maybe a literal String is better so it sticks out that this is an anamoly and an exception to the pattern
}

String genusAbbreviation = genusAndSpeciesName.substring(0,1).toLowerCase() + speciesName.split(" ")[1];
String speciesName = genusAndSpeciesName.split(" ");

return genusAbbreviation + speciesName;
```

Reply:
Sure, I'll do that